review request

femitj opened the pull request.

femitj committed:
[ft #163282570] Add login page

femitj committed:
[ft #163282570] Add login page

femitj committed:
Merge pull request #1 from femitj/ft-UI-login-page-163282570

#163282570 Add UI login page

femitj committed:
[ft #163282630] Add Sign-up page

femitj committed:
Merge pull request #2 from femitj/ft-UI-signup-page-163282630

feat(ui): user signup page ui template [delivers #163282630]

femitj committed:
[ft #163282699] Add Admin page/pages

femitj committed:
Merge pull request #3 from femitj/ft-UI-admin-page-163282699

feat(ui):  admin's Dashboard [delivers #163282699]

femitj committed:
[ft #163341827] Add user page/pages

femitj committed:
Merge pull request #4 from femitj/ft-UI-user-page-163341827

feat(ui): user's dashboard ui template [delivers #163341827]

femitj committed:
[ft #163349602] Add view candidate page to user

femitj committed:
Merge pull request #5 from femitj/ft-UI-user-views-candidates-163349602

feat(ui): User's view candidates ui template [delivers #163349602]

femitj committed:
[ft #163349753] Add view all vote lists

femitj committed:
Merge pull request #6 from femitj/ft-UI-user-views-vote-lists-163349753

feat(ui): user's views all personal votes ui template [delivers #163349753]

femitj committed:
[ft #163350051] Add user votes page

femitj committed:
Merge pull request #7 from femitj/ft-UI-user-votes-163350051

feat(ui): ui template for user votes [delivers  #163350051]

femitj committed:
[ft #163282652] User resets password

femitj committed:
Merge pull request #8 from femitj/ft-UI-user-reset-password-163282652

feat(ui): User resets password [delivers #163282652]

andela-gike commented on UI/index.html:
```diff
@@ -0,0 +1,43 @@
+                <div class='welcome-text'>
+                    <h1>decide your <span>future</span> now!</h1>
+                    <button id="signup-btn"><a id='btn' href="views/resgister.html">Sign Up</a></button>
+                </div>
```
Could you correct the spelling of your sign up url

femitj replied:
Done, changes would reflect in the next PR request... 
Thank you,

andela-gike commented on UI/styles/admin.css:
```diff
@@ -0,0 +1,193 @@
+  /* Add a red background color to navbar links on hover */
+  .navbar a:hover, .dropdown:hover .dropbtn {
+   /* background-color: green;*/
+    font-weight: 600;
```
Remove commented or unused code

femitj replied:
Implemented feedback on this already, PR request will be sent soon
Thanks alot

andela-gike commented on UI/views/admin/create-office.html:
```diff
@@ -0,0 +1,91 @@
+            <div class="navigation__wrapper">
+                <div class='container'>
+                <div><a href="../../index.html" id="header-title">Politico</a></div>
+                <!--<div class='nav-link'><a href="./.html">Create Government office</a></div>-->
```
Remove commented or unused code

femitj committed:
ch(ui): feedback response ui template [delivers #163362671]

andela-gike commented:
You can close this PR it is no longer needed

femitj committed:
ch(ui): feedback response - remove unused code ui template [delivers #163362671]

femitj committed:
Merge pull request #10 from femitj/ch-feedback-response-163362671

#163362671 Feedback response

femitj committed:
ch(ui) UI template update [#163388500]

femitj committed:
ch(ui) UI template update [#163388500]

femitj committed:
Merge pull request #11 from femitj/ch-UI-update-163388500

#163388500 UI template update

femitj committed:
feat(ui) Add politician page [finishes 163418598]

femitj committed:
Merge pull request #12 from femitj/ft-UI-politician-page-163418598

#163418598 Add politician page UI template

femitj committed:
ch(sever): install backend environment
server express, babel, travis, coveralls, nodemon, mocha test
[finishes #163446311]

femitj committed:
Merge pull request #13 from femitj/ch-setup-backend-env-163446311

#163446311 Set up back-end environment/tools (server)

femitj committed:
feat(endpoint): Creates political party api endpoint [delivers#163446235]

femitj committed:
Merge pull request #14 from femitj/ft-creates-party-api-endpoint-163446235

#163446235 Creates political party API endpoint

femitj committed:
feat(endpoint): gets ALL political party api endpoint [delivers#163452033]

femitj committed:
Merge pull request #15 from femitj/ft-getALL-party-endpoint-163452033

feat(endpoint): gets ALL political party api endpoint [delivers#16345…

femitj committed:
feat(endpoint): get a specific party endpoint[delivers#163453660]

femitj committed:
Merge pull request #16 from femitj/ft-get-specific-party-endpoint-163453660

#163453660 API endpoint that gets a specific political party

femitj committed:
feat(endpoint): edit a specific party[delivers#163454395]

femitj committed:
feat(endpoint): updates response message [delivers#163454395]

femitj committed:
Merge pull request #17 from femitj/ft-endpoint-edit-specific-party-163454395

#163454395 api endpoint that edits name of a specific party

femitj committed:
feat(endpoint) unit test for all api endpoints [delivers #163480128]

femitj committed:
Merge pull request #18 from femitj/ft-unit-tests-for-endpoints-163480128

#163480128 unit test for all api endpoints

femitj committed:
feat(ui): api endpoint that deletes a specific party [delivers #163454408]

femitj committed:
Merge pull request #19 from femitj/ft-delete-party-endpoint-163454408

#163454408 api endpoint that deletes a specific party

femitj committed:
feat(endpoint): adds api endpoint that gets a political office [delivers #163500488]

femitj committed:
Merge pull request #22 from femitj/ft-gets-a-office-endpoint-163500488

#163500488 Adds api endpoint that gets a specific political office

femitj committed:
feat(endpoint): adds api endpoint that gets ALL political offices [delivers #163500485] (#21)

femitj committed:
feat(endpoint): adds api endpoint that creates political office [delivers #163492345] (#20)

femitj committed:
publish app to heroku [finishes #163510945]

femitj committed:
Merge pull request #23 from femitj/ch-host-app-heroku-163510945

#163510945 Publish app to heroku

femitj committed:
publish app to heroku [finishes #163510945]

femitj committed:
publish app to heroku [finishes #163510945]

femitj committed:
Merge pull request #24 from femitj/ch-host-app-heroku-163510945

publish app to heroku [finishes #163510945]

femitj committed:
setting up PostgresSQL database [finishes 163514554]

femitj committed:
setting up PostgresSQL database [finishes #163514554]

femitj committed:
Merge pull request #25 from femitj/ch-set-up-database-163514554

#163514554 Set up database

femitj committed:
#163530728 creates account for users with token based authentication  (#26)

feat(endpoint): creates account for users [delivers #163530728]

* updates response spec

* updates response spec

* updates fields helper

femitj committed:
delivers #163530747 Api endpoint that logs in user (#27)

* Api endpoint that log in user [delivers #163530747]

* updates response spec & fields helper

femitj committed:
#163570754 Integrate party route to POSTGRES DB and secure routes (#28)

* feat(endpoint): Integrate party route to POSTGRES DB and secure routes [delivers #163570754]

* updated response spec

* refactoring code

femitj committed:
#163570814 Integrate office routes with POSTGRES database and authentication (#29)

* feat(ui): Integrate POSTGRES database with office routes and authentication [delivers #163570814]

* updated response spec

femitj committed:
#163530774 API endpoint that registers users as candidates for election (#30)

* feat(endpoint): API endpoint that registers politicians as candidate [delivers #163530774]

* make changes

* updated response spec

* updated response spec

femitj committed:
#163530828 Users can vote for candidate of choice   (#31)

* feat(endpoint): Users can vote for candidate of choice [delivers #163530828]

* updated response spec

* updated response spec

femitj committed:
feat(endpoint): collates and fetches specific office election result [delivers #163530920]

femitj committed:
Merge pull request #32 from femitj/ft-specific-office-vote-results-endpoint-163530920

#163530920 collates and fetches specific office election result
